src/components/cms/SkillCreator/SkillCreator.js

```diff
@@ -297,7 +297,16 @@ class SkillCreator extends Component {
                       }}
                     />
                   </Fab>
-                  <CardContent>Create a new skill</CardContent>
+                  <CardContent
+                    style={{
+                      fontSize: '1.5rem',
+                      color: 'black',
+                      textShadow: '2px 2px 8px white',
+                      fontWeight: 'bold',
+                    }}
```

Comment:
Avoid custom css. It's not a good practice.

Reply:
It's a mere change only, that's why added inline styling.

Reply:
We are using Material UI which already comes with lot of pre-defined css. We should make use of it.

Reply:
@snitin315 So Should I use Material UI or leave same for this small change

Reply:
Check if there is a predefined class/component for same , if found use it.

Reply:
ok